test/testdrive/show.td

```diff
@@ -199,33 +199,34 @@ VIEWS        TYPE  QUERYABLE
 names_view   USER  true
 
 > SHOW FULL SOURCES FROM mz_catalog
-VIEWS                             TYPE   MATERIALIZED
------------------------------------------------------
-mz_arrangement_sharing            SYSTEM true
-mz_arrangement_sizes              SYSTEM true
-mz_avro_ocf_sinks                 SYSTEM true
-mz_catalog_names                  SYSTEM true
-mz_dataflow_channels              SYSTEM true
-mz_dataflow_operator_addresses    SYSTEM true
-mz_dataflow_operators             SYSTEM true
-mz_kafka_sinks                    SYSTEM true
-mz_materialization_dependencies   SYSTEM true
-mz_materialization_frontiers      SYSTEM true
-mz_materializations               SYSTEM true
-mz_peek_active                    SYSTEM true
-mz_peek_durations                 SYSTEM true
-mz_scheduling_elapsed             SYSTEM true
-mz_scheduling_histogram           SYSTEM true
-mz_scheduling_parks               SYSTEM true
-mz_view_foreign_keys              SYSTEM true
-mz_view_keys                      SYSTEM true
+VIEWS                                TYPE   MATERIALIZED
```

Comment:
This confused me, as I didn't expect this to be a `VIEW`, but it looks like it is a bug with `SHOW SOURCES` instead?

Reply:
I have no idea, actually. It seems fine locally.
```
materialize=> show full sources from mz_catalog;
             SOURCES             |  TYPE  | MATERIALIZED 
```